apps/api/prisma/schema.prisma

```diff
@@ -306,18 +306,63 @@ model Folder {
 }
 
 model Document {
-  guid                 String   @id @default(uuid())
+  guid                 String             @id @default(uuid())
   name                 String
-  folder               Folder   @relation(fields: [folderId], references: [id])
+  folder               Folder             @relation(fields: [folderId], references: [id])
   folderId             Int
   blobStorageContainer String?
   blobStoragePath      String?
-  status               String   @default("awaiting_upload")
-  redacted             Boolean  @default(false)
-  isDeleted            Boolean  @default(false)
-  createdAt            DateTime @default(now())
-  fileSize             Int      @default(0)
+  status               String             @default("awaiting_upload")
+  redacted             Boolean            @default(false)
+  isDeleted            Boolean            @default(false)
+  createdAt            DateTime           @default(now())
+  fileSize             Int                @default(0)
   fileType             String?
```

Comment:
We will need to remove these columns, because they are redundant to the ones in the Metadata table

Reply:
yeah I agree will need to do that slowly don't wan't to break anything